pkg/store/gcs.go

```diff
@@ -352,25 +353,23 @@ func (s *GCSStore) blockSeries(ctx context.Context, b *gcsBlock, matchers []labe
 	defer indexr.Close()
 	defer chunkr.Close()
 
-	begin := time.Now()
-
+	blockPrepareBegin := time.Now()
```

Comment:
I removed all the spans from here, since for most queries the trace was too massive.